### PR TITLE
Cleanup of utils.js

### PR DESCRIPTION
**Pull request by chrischrischris**
Move funcs used in only one place out of utils

Trying to keep utils as small as possible.

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://c3-utilcleanup--milo--adobecom.hlx.page/?martech=off

### CODE REVIEW

File: libs/blocks/gnav/gnav.js [hunk 1]
```diff
@@ -31,6 +30,14 @@ const debounce = (func, timeout = 300) => {
   };
 };
 
+function getBlockClasses(className) {
```
Comment (auniverseaway) [inline libs/blocks/gnav/gnav.js:33]: Pretty sure this is obsolete now. I can look into with a separate PR.

File: libs/utils/utils.js [hunk 2]
```diff
@@ -564,6 +547,9 @@ export const analyticsDecorateList = (li, idx) => {
   link.setAttribute('daa-ll', `${analyticsGetLabel(label)}-${idx + 1}`);
 };
 
+export const utf8ToB64 = (str) => window.btoa(unescape(encodeURIComponent(str)));
```
Comment (vhargrave) [inline libs/utils/utils.js:550]: nit: we should probably switch out the half deprecated methods "unescape" and "escape" here with something like: 
```
export const utf8ToB64 = (str) => window.btoa(decodeURI(encodeURIComponent(str)));
export const b64ToUtf8 = (str) => decodeURIComponent((window.atob(str)).replace('\\x', '%'));
```

**Review by vhargrave — APPROVED** [review]
looking good !

File: libs/utils/utils.js (hunk 2 above)
Comment (chrischrischris) [reply libs/utils/utils.js:536]: @vhargrave probably a good idea, but I don't want to risk breaking any current encodings.  Will take some testing so I'll have to get back to this at another time.

**Pull request merged**